bemani/backend/museca/museca1plus.py

```diff
@@ -197,6 +245,23 @@ def enable_event(eid: int) -> None:
         info.add_child(Node.string('param_str_3', 'available_ex: 1'))
         info.add_child(Node.string('param_str_4', 'available_ex: 1'))
         info.add_child(Node.string('param_str_5', 'available_ex: 1'))
+        
+        if self.omnimix:
```

Comment:
What does this whole section do? Your PR has absolutely no description.

Reply:
As the comment in the block above it explains: `# Makes special missions available on grafica that have them.`
I just tacked one on for a museca plus mission. I figured it was kind of self explanatory, sorry.

Reply:
Also, an `extend/info` node can only have 5 `param_num` and `param_str` elements, which is why added a second info node.